app/DTOs/PrivateNetworkDTO.php

```
@@ -0,0 +1,56 @@
+
+        $prefix = (int) $parts[1];
+
+        if ($prefix < 0 || $prefix > 32) {
+            return null;
+        }
+
+        if (filter_var($parts[0], FILTER_VALIDATE_IP, FILTER_FLAG_IPV4) === false) {
+            return null;
+        }
+
+        return Cidr::canonical($cidr);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\Cidr' in method 'normalizeCidr'. (undefined)

(StaticAccess)